integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,51 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupTeam"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .id + \"/home\""
+            member: (.members | map(.user.username | tostring))
```

Comment:
Why are you filtering username instead of email?

Reply:
Fixed, there was no email in the docs but when I tried it on postman it was in the data so I have updated it